.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ jobs:
             wget https://storage.googleapis.com/istio-build/proxy/envoy-debug-5d798021523fa1e587fa45d48b4ce38e25967ed3.tar.gz -O envoy.tar.gz
             tar xvzf envoy.tar.gz
             ln -sf ~/envoy/usr/local/bin/envoy /go/src/istio.io/istio/pilot/proxy/envoy/envoy
-      - run: cd /go/src/istio.io/istio; go test ./pilot/...
+      - run: cd /go/src/istio.io/istio; go test ./pilot/... ./broker/... ./security/...
```

Comment:
mixer missing ?

Reply:
because it doesn't compile with go test.. it requires bazel :(. We cant run bazel in this small docker container